src/Sarif.UnitTests/Readers/UriConverterTests.cs

```diff
@@ -104,8 +104,9 @@ private void TestConverter(string inputUri, string expectedUri)
   ""runs"": [
     {
       ""tool"": {
-        ""name"": null
+        ""name"": ""DefaultTool""
       },
+      ""columnKind"": ""utf16CodeUnits"",
```

Comment:
>""columnKind"": ""utf16CodeUnits"" [](start = 6, length = 34)

It might not longer interest or surprise you to hear that I don't like updating inlined literals when maintaining tests.

Reply:
I've started the journey. Please see my JSchema PR [https://github.com/Microsoft/jschema/pull/91](https://github.com/Microsoft/jschema/pull/91). Would you please review this so when I fix #88 and #89 I can build on the new infrastructure.

---
In reply to: [246164297](https://github.com/Microsoft/sarif-sdk/pull/1204#discussion_r246164297) [](ancestors = 246164297)

Reply:
Will take a look. The most recent SARIF SDK FileDiffing base class provides an end-to-end pattern for this, did you happen to look at it?

Reply:
I didn't. I was going by your description in email about what you were after. But I will take a look.

I expect the bulk of my change to survive that examination because it's mostly about ripping literals out into files and renaming a few variables, but I might get a surprise.

---
In reply to: [246508060](https://github.com/Microsoft/sarif-sdk/pull/1204#discussion_r246508060) [](ancestors = 246508060)

Reply:
You may well be right. Two things: please take a quick look and let me know whether the PR even moves in the right direction. If not, I'll discard it. In either case, I accept your offer to rough out the file diff solution -- whether or not the PR is a useful base for you to do that.

Finally, if we have infrastructure shared across 5 repos, we really need a separate repo to hold it, to include as a submodule in the 5 clients.

---
In reply to: [246550880](https://github.com/Microsoft/sarif-sdk/pull/1204#discussion_r246550880) [](ancestors = 246550880,246525111,246508060)